src/main/java/io/nozemi/runescape/content/magic/NormalTeleports.java

```diff
@@ -33,4 +33,94 @@ public void homeTeleport(Player player, int option, int item) {
             player.teleport(new Tile(2606, 3102));
         }
     }
+    @Button(parentId = 218, childId = 16)
+    public void varrockTeleport(Player player, int option, int item) {
+        Optional<MyTeleports> teleport = TeleportEffectChainHandler.getTeleport("VARROCK");
+
+        if(teleport.isPresent()) {
+            player.teleport(new Tile(2722, 3484), teleport.get()); //Yanille
```

Comment:
Does this teleport you to Dodian home? Because the teleports are meant to be the same as in current Dodian.

Reply:
Ancient are the same spells as dodian. modern is disabled